allennlp/predictors/dialog_qa.py

```diff
@@ -0,0 +1,56 @@
+import json
+from overrides import overrides
+
+from allennlp.common.util import JsonDict
+from allennlp.data import DatasetReader, Instance
+from allennlp.data.tokenizers.word_splitter import SpacyWordSplitter
+from allennlp.predictors.predictor import Predictor
+from allennlp.models import Model
+
+
+@Predictor.register('dialog_qa')
+class DialogQAPredictor(Predictor):
+    def __init__(self, model: Model, dataset_reader: DatasetReader) -> None:
+        super().__init__(model, dataset_reader)
+        self._tokenizer = SpacyWordSplitter(language='en_core_web_sm')
+
+    def predict(self, jsonline: str) -> JsonDict:
+        """
+        Make a machine comprehension prediction on the supplied input.
+       -------
```

Comment:
This is why the build is failing - this needs to be formatted like the rest of our docstrings (you probably meant this to be a "Returns" block, and the spacing here is off).

Reply:
And, even though I approved the PR, I can't merge it until the build passes, just FYI.